### PR TITLE
SISRP-26739 - 7.2: Student Resources: eForm Links - re-organize card

### PR DESCRIPTION
**Pull request by dfkaye**
https://jira.berkeley.edu/browse/SISRP-26739

### CODE REVIEW

File: src/assets/templates/widgets/student_resources.html [hunk 1]
```diff
@@ -7,28 +7,95 @@
...
-          </div>
-        </li>
-        <li data-ng-if="studentResources.withdrawFromSemesterAdd.url && !isLawStudent">
-          <a data-ng-href="{{studentResources.withdrawFromSemesterAdd.url}}" data-ng-attr-title="{{studentResources.withdrawFromSemesterAdd.title}}">Withdraw from a Semester</a>
-        </li>
-        <li data-ng-if="studentResources.veteransBenefitsAdd.url">
-          <a data-ng-href="{{studentResources.veteransBenefitsAdd.url}}" data-ng-attr-title="{{studentResources.veteransBenefitsAdd.title}}">Veterans Benefits Intake</a>
-        </li>
-      </ul>
+      <div >
```
Comment (lyttam) [inline src/assets/templates/widgets/student_resources.html]: @dfkaye Extra space in `<div >`

File: app/models/campus_solutions/student_resources.rb [hunk 2]
```diff
@@ -19,8 +19,13 @@ def get
...
         { feed_key: :emergency_loan_form_add, cs_link_key: 'UC_CX_GT_FAEMRLAON_ADD' },
         { feed_key: :emergency_loan_form_view, cs_link_key: 'UC_CX_GT_FAEMRLAON_VIEW' },
-        { feed_key: :withdraw_from_semester_add, cs_link_key: 'UC_CX_SRWITHDRL_ADD' },
+        { feed_key: :higher_degrees_committee_form, cs_link_key: 'UC_CX_GT_AAQEAPPLIC_ADD' },
+        { feed_key: :special_enrollment_petition, cs_link_key: 'UC_CX_GT_SRSEP_ADD' },
+        { feed_key: :submit_degree_candidacy_form, cs_link_key: 'UC_CX_GT_SRDCR_ADD' },
+        { feed_key: :update_pending_forms, cs_link_key: 'UC_CX_GT_STUDENT_UPDATE' },
         { feed_key: :veterans_benefits_add, cs_link_key: 'UC_CX_GT_SRVAONCE_ADD' },
+        { feed_key: :view_submitted_forms, cs_link_key: 'UC_CX_GT_STUDENT_VIEW'},
+        { feed_key: :withdraw_from_semester_add, cs_link_key: 'UC_CX_SRWITHDRL_ADD' }
```
Comment (redconfetti) [inline app/models/campus_solutions/student_resources.rb]: @dfkaye - You could have left an extra comma on the end of this element. This isn't Javascript you know ;)

**Comment by redconfetti** [comment]
Looks good to me @dfkaye, but I cannot merge while this PR has a Git conflict.

**Comment by dfkaye** [comment]
@redconfetti - looks ready now - conflict resolved, travis passing...

**Pull request merged**